.github/actions/gam-canary-rollback/canary-probe.mjs

```
@@ -0,0 +1,156 @@
+//   EXPECT_HEADER      "Name: value-substr" that MUST appear in a response header (e.g. a version/etag).
+//   EXPECT_STATUS      expected HTTP status (default: any 2xx).
+//   MAX_ATTEMPTS       retries before declaring a verdict (default 6 — a fresh deploy may warm slowly).
+//   RETRY_DELAY_MS     delay between attempts (default 10000).
+//   TIMEOUT_MS         per-request timeout (default 15000).
+//
+// Exit: 0 = healthy (deploy PROVEN live) · 1 = regressed (caller should ROLLBACK)
+//       · 2 = usage/config error (REFUSE — nothing was probed) · 3 = refuse (probed, but every
+//       attempt failed to produce a determinate answer — caller should ROLLBACK, same as regressed)
+// Emits a JSON receipt to stdout on the last line (parse with `tail -1`), human lines to stderr.
+
+const env = process.env;
```

ESLint

[error] 'process' is not defined.

(no-undef)